openduck-py/openduck_py/routers/voice.py

```diff
@@ -67,8 +65,30 @@
 processes = {}
 
 
-def _transcribe(audio_data):
-    return whisper_model.transcribe(audio_data)["text"]
+async def _transcribe(audio_data: np.ndarray) -> str:
+    assert audio_data.dtype == np.float32
+    wav_io = BytesIO(audio_data.tobytes())
+    wav_data = wav_io.getvalue()
+
+    # Send the POST request to the endpoint
+    url = "http://openduck_ml_1:8001/ml/transcribe"
```

Comment:
This URL is only for docker dev, need to change it for prod

Reply:
Let's use a value from `.env` and put a placehodler in `.env.example`